packages/viewtoggle/package.json

```diff
@@ -26,7 +26,7 @@
     "prop-types": "^15.7.2"
   },
   "peerDependencies": {
-    "@pluralsight/ps-design-system-normalize": "*",
+    "@pluralsight/ps-design-system-normalize": "v3.1.4",
```

Comment:
I'm pretty sure these numbers don't need and can't have the `v` prefix.  Do you know otherwise?

Reply:
oops. it should be a ^